### PR TITLE
Merge [feature/add-documentation] into [master]

### PR DESCRIPTION
**Pull request by omadoyeabraham**
- Added README.md
- bumped prisma-engine
- added dotenv to build.rs
- updated calls `execute` and `execute_all` functions in `prisma.template.rs` to include `None` as the trace_id (required because of bump to prisma-engine)

### CODE REVIEW

**Comment by seunlanlege** [comment]
Dotenv is actually a really brilliant idea, but instead of a path dependency, let the Readme use a git dependency.

**Comment by omadoyeabraham** [comment]
@seunlanlege The README has been updated to use a git dependency, and an error message has been added to the dotenv use in build.rs as well.

File: README.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ An auto-generated and type-safe query builder  in rust for [Prisma](https://pris
...
    ...
-    prisma-client = { path = "./prisma-client-rs" }
+    prisma-client = { git = "https://github.com/polytope-labs/prisma-client-rs" }
     serde = { version = "1.0", features = ["serde_derive"] }
     ```
-4. Create a .env in your project root and add the following:
+3. Create a .env in your project root and add the following:
    ```
-   PRISMA_SCHEMA="../prisma/schema.prisma"
+   PRISMA_SCHEMA={PATH_TO_SCHEMA_FILE_HERE}
```
Comment (seunlanlege) [inline README.md]:
```suggestion
   PRISMA_SCHEMA=PATH/TO/SCHEMA/FILE/HERE
```

File: README.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ An auto-generated and type-safe query builder  in rust for [Prisma](https://pris
...
 folder should be siblings.
-2. Clone the prisma-client-rs repository into your project root using `git clone git@github.com:polytope-labs/prisma-client-rs.git`.
-3. Update your dependencies in the `Cargo.toml` file to include `prisma-client-rs` and `serde`, ensure the path points to the location of the cloned prisma-client-rs repository.
+
+2. Update your dependencies in the `Cargo.toml` file to include `prisma-client-rs` and `serde`.
     ```toml
     [dependencies]
    ...
-    prisma-client = { path = "./prisma-client-rs" }
+    prisma-client = { git = "https://github.com/polytope-labs/prisma-client-rs" }
```
Comment (seunlanlege) [inline README.md]:
```suggestion
    prisma-client = { git = "https://github.com/polytope-labs/prisma-client-rs", branch = "master" }
```

**Pull request merged**